lib/xml.h

```diff
@@ -31,4 +32,12 @@ SUPPRESS_WARNING_CLANG_POP
 SUPPRESS_WARNING_CLANG_POP
 SUPPRESS_WARNING_CLANG_POP
 
+inline static tinyxml2::XMLError xml_LoadFile(tinyxml2::XMLDocument& doc, const char* filename)
```

Comment:
This should be used in all other cases as well but we need to add tests for all of those before we do that.